app/templates/project/conversations.hbs

```diff
@@ -0,0 +1,62 @@
+<div class="container">
+  <div class="conversations">
+    <section>
+      <aside>
```

Comment:
I have yet to add the styling for `.conversations` but it will differ from the `.panel` styling. The left sidebar will be a scrolling list of messages and the right side will contain two panes: left for the message thread, right for the information about the target user or project. In total, then, the UI will appear to have three panes, but the right two panes are basically the `outlet` for the individual conversation.